packages/stargate/src/stargateclient.ts

```diff
@@ -54,14 +60,14 @@ export interface BroadcastTxFailure {
   readonly code: number;
   readonly transactionHash: string;
   readonly rawLog?: string;
-  readonly data?: Uint8Array;
+  readonly data?: readonly IMsgData[];
```

Comment:
Ideally we has this, such that there is always one entry per message. But I see that the protobuf type system makes this hard to get.

```suggestion
  readonly data: readonly IMsgData[];
```